libraries/elixir/README.md

```diff
@@ -0,0 +1,58 @@
+Elixir library for Standard Webhooks
+
+Need to handle inbound webhooks in Elixir? Check out [webhoox](https://github.com/cpursley/webhoox).
+
+# Example
+
+Verifying a webhook payload:
+
+```elixir
+StandardWebhooks.verify(plug_conn, webhook_payload, webhook_secret)
```

Comment:
Sorry for coming late in the party, but I'm not a fan of it having a different API to the rest of the libs, I think there's value in having all the libs be very similar.

On a personal level I also prefer the other way in general. Creating the other structure should be essentially zero cost, and it centralizes the parsing of the secret to one place so you know whether your secret is malformed before it's time to verify, but also you get to reuse this parsed structure.

Though it's really more about the APIs being the same across all of the libs.